google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/ITBigQueryWriteManualClientTest.java

```diff
@@ -468,6 +433,172 @@ public void testJsonStreamWriterWithDefaultSchema()
     }
   }
 
+  @Test
+  public void testMultipleJsonStreamWriterWithDefaultSchema()
```

Comment:
I think we only need one test with schema and one test without schema.

Reply:
Removed multiple test.